project2.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="project2.css" type="text/css">
+  <link href="https://font.googleapis.com/css?family=Lobster" rel="stylesheet" type="text/css">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css"
+  integrity="sha512-KfkfwYDsLkIlwQp6LFnl8zNdLGxu9YAA1QvwINks4PhcElQSvqcyVLLD9aMhXd13uQjoXtEKNosOWaZqXgel0g==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+  <link rel="stylesheet" href="https://fonts.sandbox.google.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@48,400,0,0" />
+  <title>Setup and Mobile Version Skeleton</title>
+</head>
+
+<body>
+  <header>
+<div class="grid-container">
+  
+    <div class="one">
+        <h1>Multi-Post Stories</h1>
+        
+       <button class="button0">Facebook</button>
+       <button class="button00">Full Stack Dev</button>
+       <button class="button01">2015</button>
```

Comment:
Don't forget the usage of `<ul>` and `<li>` for this **list** of attributes :fire: